.github/workflows/build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           environment-file: environment.yml
       - run: ./tools/extract_version.sh
       - run: ./tools/macos_install_certificates.sh
-        if: ${{ github.event_name != 'pull_request' }}
+        # if: ${{ github.event_name != 'pull_request' }}
```

Comment:
I don't think secrets get passed to PRs so commenting this out isn't enough. You'd need to be pushing to a branch on this repo for example rather than your fork

Reply:
oooohhhh